pyproject.toml

```diff
@@ -49,6 +49,8 @@ Babel = { version = "^2.9.1", optional = true }
 aiohttp-socks = { version = "^0.5.5", optional = true }
 # Redis
 aioredis = { version = "^2.0.0", optional = true }
+# Mongodb
+motor = { version = "^2.5.1", optional = true }
```

Comment:
This dependency should also be added to `[tool.poetry.extras]` section: `mongo = ["motor"]`
After that add to the Makefile in the target `install` new argument `-E mongo`